garage.com/src/test/java/garage/NewTest.java

```diff
@@ -0,0 +1,159 @@
+package garage;
+import static org.testng.Assert.assertEquals;
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NewTest {
+	
+	@Test
+	public void f1() {
```

Comment:
Use descriptive names for your tests. One day you will be looking through thousands of test results. If they were all named "f1", "test1", etc then it will be difficult to know what those tests do.